### ISSUE TITLE
Processes/Collections: Return supported back-ends as a list?

### ISSUE THREAD

**Issue by m-mohr** [Open-EO/openeo-aggregator#10]
Would it make sense to return a list of back-ends in the JSON metadata that support a certain collection / process? 

For example:
```js
{
  "id": "mask",
  "backends": ["eodc", "vito"], // list id that is exposed in capabilities?
  ...
}
```

This would allow us to add a "compliance" matrix to the documentation, which a user can get a quick overview from what is supported where.

For example (just listed random names with randomly placed yes/no icons):
![image](https://user-images.githubusercontent.com/8262166/136016712-5527bb29-78c2-48b9-b5a2-ad1eaf200e59.png)

This should be relatively easy to implement both server- and client-side, I think.

Thoughts? @jdries @soxofaan

**Comment by soxofaan** [comment]
yes, it would be straightforward to add that metadata.
It's just a matter of standardizing the field name and format.

FYI, for collections provided by multiple back-ends there is already a `summaries` entry at key `"provider:backend"`, e.g. see this test:
https://github.com/Open-EO/openeo-aggregator/blob/8e5056900adf31f8fb2b90eaec8fa2427ed98900/tests/test_backend.py#L146

(because there is no pure id-based overlap yet between VITO and EODC, this is not yet visible in production)

**Comment by m-mohr** [comment]
I don't think we need to standardize this yet, this can be a platform-specific API extension and could be implemented right away.

**Comment by soxofaan** [comment]
I don't meant to standardize on level of openEO API (yet), just agree between aggregator and client/docs/website on a scheme that's reasonably future proof

**Comment by m-mohr** [comment]
I've updated the example above to `"backends": ["eodc", "vito"], // list id that is exposed in capabilities?` - It could be any name and I don't have a strong preference, but I think what I would list in the array are the back-end IDs from the capabilities. Other than that, I think we could continue with the implementation. It's new and we can just implement it client-side against whatever is returned by the aggregator.

**Comment by m-mohr** [comment]
URL as id is also a good idea.

**Comment by m-mohr** [comment]
Some additional ideas I had this morning: https://github.com/openEOPlatform/architecture-docs/issues/85#issuecomment-945554415

Especially: In STAC world "provider:backend" is a bit strange, better use something project-specific, e.g. "openeo:backend". This is a bit different from what I told before, sorry.

**Comment by soxofaan** [comment]
I would also favor namespaced scheme like `extension:field_name`  , that feels more future proof,
but I'd let you choose as you have more experience in the STAC ecosystem

**Comment by soxofaan** [comment]
this is apparently already fixed/covered under #76/#77/#95
